libraries/cli/include/cli/default_config.hpp

```diff
@@ -8,14 +8,17 @@ const char *default_json = R"foo({
   "network_address": "0.0.0.0",
   "network_tcp_port": 10002,
   "network_udp_port": 10002,
-  "network_simulated_delay": 0,
   "network_transaction_interval": 100,
-  "network_bandwidth": 40,
   "network_ideal_peer_count": 10,
   "network_max_peer_count": 50,
   "network_sync_level_size": 10,
   "network_packets_processing_threads": 10,
   "deep_syncing_threshold" : 10,
+  "bandwidth_throttle_period_seconds" : 10,
+  "max_allowed_total_packets_size" : 52428800,
+  "max_allowed_total_packets_count" : 500,
+  "max_allowed_same_type_packets_size" : 10485760,
+  "max_allowed_same_type_packets_count" : 100,
```

Comment:
I find it too low, we have now millions of unprocessed vote packets

Reply:
yeah I dont really know what limits should I set there, currently it is (per 10sec):

- 10k packets count of the same type
- 10MB packets size of the same type
- 25k packets count of the all type
- 15MB packets size of the all type